exercises/classes/makeCircle/index.ts

```diff
@@ -0,0 +1,13 @@
+class Circle {
```

Comment:
Let's export all our solutions.
`export class Circle`

Reply:
Updated in a new commit